mozilla_django_oidc/urls.py

```diff
@@ -16,3 +18,16 @@
         name='oidc_authentication_init'),
     url(r'^logout/$', views.OIDCLogoutView.as_view(), name='oidc_logout'),
 ]
+
+
+if 'test' in sys.argv:
+    # If we're running tests, it helps to have a "home page" we can test
+    # against that doesn't trigger authentication things.
+    from django.http import HttpResponse
+
+    def fakeview(req):
+        return HttpResponse('Win!')
+
+    urlpatterns.append(
+        url(r'^mdo_fake_view/$', fakeview, name='mdo_fake_view')
+    )
```

Comment:
What if we subclass ``TestCase`` and add an extended urlconf same way you do it here. That means:

* Import and extend urls under /test
* Configure TestCase to use the patched urlconf

https://docs.djangoproject.com/en/1.11/topics/testing/tools/#urlconf-configuration

If it feels to much of an overhead let's leave it as it is.